### PR TITLE
Get antenna montioring values in GNSS collector 

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

File: pkg/collectors/devices/gps_ubx.go [hunk 1]
```diff
@@ -20,21 +20,38 @@ type GPSNav struct {
 	TimestampStatus string `json:"timestampStatus" fetcherKey:"navStatusTimestamp"`
 	TimestampClock  string `json:"timestampClock" fetcherKey:"navClockTimestamp"`
 	GPSFix          string `json:"GPSFix" fetcherKey:"gpsFix"`
+	TimestampMon    string `json:"timestampMon" fetcherKey:"monTimestamp"`
+	AntBlockID      []int  `json:"antBlockId" fetcherKey:"antBlockId"`
```
[the remark this answers is not in the document]
Comment (crwr45) [reply pkg/collectors/devices/gps_ubx.go:24]: Looking at the rest of this the `Ant` fields should all be in a struct of their own, with one instance per block. That seems to be the natural representation of the data.

**Review by crwr45 — CHANGES_REQUESTED** [review]

File: pkg/collectors/devices/gps_ubx.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (nocturnalastro) [reply pkg/collectors/devices/gps_ubx.go]: I agree :)

**Review by crwr45 — APPROVED** [review]
That's much better, more refined and understandable. lgtm, though we need to consider how it affects the analysers.

**Comment by nocturnalastro** [comment]
> That's much better, more refined and understandable. lgtm, though we need to consider how it affects the analysers.

Output format is still the same as it is currently as I unpack the values when presenting them [here](https://github.com/redhat-partner-solutions/vse-sync-testsuite/pull/49/files#diff-1a0886da543185f28fc547ad96dd0e84b69178d13556d7be42e050c7d7716f84R48-R51)

**Pull request merged**